memory/memory_index.py

```
@@ -0,0 +1,981 @@
+                    tags = excluded.tags,
+                    source = excluded.source,
+                    content_hash = excluded.content_hash,
+                    file_path = excluded.file_path,
+                    raw_text = excluded.raw_text
+            """, (
+                memory_id, entity_id, entity_type, folder, memory_type,
+                importance, timestamp, last_accessed, access_count,
+                tags_json, source_json, chash, file_path, base_dir, raw_text,
+            ))
+
+            # 同步 FTS 索引（用分词后的文本）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)